packages/webpack/test/e2e-main.spec.js

```diff
@@ -12,6 +12,7 @@ test.before(async (t) => {
 })
 
 test('webpack/main - dist shape', (t) => {
+  // Note: The fixture contains a few cases of resources that are emitted by default. This test is to ensure that the default behavior is not changed for the main app (hence the oddly named html file and the svg file from a dependency are emitted). resources from dependencies are prevented from getting emitted, so the other html resource is not showing up in dist. For the svg to be there a loader had to be added to the webpack config.
```

Comment:
I'd probably retitle the test to something based on this description.  Also the last sentence could be transposed. "A loader needs to be added to the webpack config.."